apps/dashboard/src/components/auth/inbox-playground.tsx

```diff
@@ -1,222 +1,206 @@
-import { useEnvironment } from '@/context/environment/hooks';
 import { useTriggerWorkflow } from '@/hooks/use-trigger-workflow';
-import { zodResolver } from '@hookform/resolvers/zod';
-import { useEffect, useState } from 'react';
-import { useForm } from 'react-hook-form';
-import { RiNotification2Fill } from 'react-icons/ri';
+import { useState } from 'react';
+import { RiFileCopyLine } from 'react-icons/ri';
+import { Notification5Fill } from '@/components/icons';
 import { useNavigate } from 'react-router-dom';
-import { z } from 'zod';
 import { ONBOARDING_DEMO_WORKFLOW_ID } from '../../config';
-import { useAuth } from '../../context/auth/hooks';
 import { useTelemetry } from '../../hooks/use-telemetry';
-import { useInitDemoWorkflow } from '../../hooks/use-init-demo-workflow';
 import { ROUTES } from '../../utils/routes';
 import { TelemetryEvent } from '../../utils/telemetry';
 import { Button } from '../primitives/button';
-import { InlineToast } from '../primitives/inline-toast';
-import { showErrorToast, showSuccessToast } from '../primitives/sonner-helpers';
+import { showToast } from '../primitives/sonner-helpers';
 import { UsecasePlaygroundHeader } from '../usecase-playground-header';
-import { CustomizeInbox } from './customize-inbox-playground';
+import { ToastIcon } from '../primitives/sonner';
+import { useOrganization } from '@clerk/clerk-react';
 import { InboxPreviewContent } from './inbox-preview-content';
+import { useInitDemoWorkflow } from '@/hooks/use-init-demo-workflow';
+import { useEnvironment } from '@/context/environment/hooks';
 
-export interface ActionConfig {
-  label: string;
-  redirect: {
-    target: string;
-    url: string;
-  };
-}
-
-export interface InboxPlaygroundFormData {
-  subject: string;
-  body: string;
-  primaryColor: string;
-  foregroundColor: string;
-  selectedStyle: string;
-  openAccordion?: string;
-  primaryAction: ActionConfig;
-  secondaryAction: ActionConfig | null;
-  enableTabs?: boolean;
+const PLAYGROUND_CONFIG = {
+  title: 'The <Inbox/> your app deserves',
+  description: "This is what your users will see. Try sending a message. We've prefilled it for you.",
+  currentStep: 2,
+  totalSteps: 4,
+} as const;
+
+function generateCurlCommand(userId: string): string {
+  return `curl -X POST ${window.location.origin}/api/v1/events/trigger \\
+  -H "Content-Type: application/json" \\
+  -H "Authorization: Bearer YOUR_API_KEY" \\
+  -d '{
+    "name": "${ONBOARDING_DEMO_WORKFLOW_ID}",
+    "to": {
+      "subscriberId": ${JSON.stringify(userId)}
+    },
+    "payload": {}
+  }'`;
```

Comment:
I thin i've already seen this snippet somewhere in the this PR, let's make sure it's reusable